File: 5. Courses/1. CS50X/tests/lec6/sentimental-mario-more/mario.py
from cs50 import get_int
Height = 0
while Height < 1 or Height > 8:
    Height = get_int("Height: ")

for i in range(Height):
    for j in range(Height - i - 1):
        print(" ", end="")
    for k in range(i + 1):
        print("#", end="")
    print("  ", end="")
    for n in range(i + 1):
        print("#", end="")
    print()

File: 5. Courses/1. CS50 Intro To CS/lec1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>
int main(void)
{
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1);

    for (int i = 1; i <= h; i++)
    {
        for (int j = 1; j <= h - i; j++)
        {
            printf(" ");
        }
        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }

        // two spaces:
        printf("  ");

        // second triangle:

        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}
